spec/transaction/paths.yaml

```diff
@@ -580,7 +580,8 @@ paths:
             example: yLLiDqX2OL7mcIMTjob60A==
       responses:
         "200":
-          description: Asset file uploaded part successfully
+          description: Asset file uploaded part successfully (Response as no content).
```

Comment:
Again, probably "asset file part uploaded" instead of "asset file uploaded part"?
And "response **h**as no content".

Reply:
Done, thanks.